wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -156,6 +161,57 @@ def page_permissions(context, page):
     return _get_user_page_permissions(context).for_page(page)
 
 
+@register.simple_tag
+def is_page(obj):
+    """
+    Usage: {% is_page obj as is_page %}
+    Sets the variable 'is_page' to True if the given object is a Page instance,
+    False otherwise. Useful in shared templates that accept both Page and
+    non-Page objects (e.g. snippets with the optional features enabled).
+    """
+    return isinstance(obj, Page)
+
+
+@register.simple_tag(takes_context=True)
+def admin_edit_url(context, obj, user=None):
```

Comment:
I think some of it may just be the fact that `AdminURLFinder` was added more recently (in Wagtail's terms) than many other features, to make audit logging easier I believe